CHANGELOG.md

```diff
@@ -5,7 +5,39 @@ This file documents any relevant changes done to ViUR server since version 2.
 
 ## [develop] - Current development version
 
-(coming soon)
+### Added
+- Pagesize to sha256 cache key
+- Providing less or equal and more or equal comparision filters in queries. ([#77](https://github.com/viur-framework/server/pull/77))
+- `_queue` Paramater to deferred calls. This allows to enqueue deferred tasks to other, custom defined task-queues. ([#133](https://github.com/viur-framework/server/pull/133))
+- Default template for captcha-bone ([#112](https://github.com/viur-framework/server/pull/112))
+- getAvailableRootNodes stub ([#113](https://github.com/viur-framework/server/pull/113))
+
+### Changed
+- Renamed folder "icons" to "embedsvg" ([#85](https://github.com/viur-framework/server/pull/85))
+- Allow to select multiple kinds in DB import/export ([#102](https://github.com/viur-framework/server/pull/102))
+- Force SSL by default ([#132](https://github.com/viur-framework/server/pull/132))
+- Limit the timewindow for second factor to 10 minutes ([#129](https://github.com/viur-framework/server/pull/129))
+- Polishing IndexManager ([#76](https://github.com/viur-framework/server/pull/76))
+- Use `.get()` for _servingurl_-test ([#114](https://github.com/viur-framework/server/pull/114))
```

Comment:
Change this to "Improved internal handling of tests on servingurls", that is easier to understand.